deploy/crds/sriovnetwork.openshift.io_sriovnetworknodepolicies_crd.yaml

```diff
@@ -43,6 +43,12 @@ spec:
             isRdma:
               description: RDMA mode. Defaults to false.
               type: boolean
+            linkType:
+              description: NIC Link Type. Allowed value "ETH", "IB".
+              enum:
+                - ETH
```

Comment:
Shall we also allow lower case and full names such as `eth`, `ether`, `ib`, `infiniband` etc?

Reply:
maybe just ib and eth so we will suppot (IB/ETH/ib/eth)